bot/kodiak/test_evaluation.py

```diff
@@ -280,25 +332,152 @@ def review() -> PRReview:
     )
 
 
+def create_review() -> PRReview:
+    return PRReview(
+        state=PRReviewState.APPROVED,
+        createdAt=datetime(2015, 5, 25),
+        author=PRReviewAuthor(login="ghost", permission=Permission.WRITE),
+    )
+
+
 @pytest.fixture
 def context() -> StatusContext:
     return StatusContext(context="ci/api", state=StatusState.SUCCESS)
 
 
+def create_context() -> StatusContext:
+    return StatusContext(context="ci/api", state=StatusState.SUCCESS)
+
+
 @pytest.fixture
 def check_run() -> CheckRun:
     return CheckRun(name="WIP (beta)", conclusion=CheckConclusionState.SUCCESS)
 
 
+def create_check_run() -> CheckRun:
+    return CheckRun(name="WIP (beta)", conclusion=CheckConclusionState.SUCCESS)
+
+
 @pytest.fixture
 def review_request() -> PRReviewRequest:
     return PRReviewRequest(name="ghost")
 
 
+def create_review_request() -> PRReviewRequest:
+    return PRReviewRequest(name="ghost")
+
+
 def test_config_fixtures_equal(config_str: str, config: V1) -> None:
     assert config == V1.parse_toml(config_str)
 
 
+def create_repo_info() -> RepoInfo:
+    return RepoInfo(
+        merge_commit_allowed=True,
+        rebase_merge_allowed=True,
+        squash_merge_allowed=True,
+        is_private=False,
+    )
+
+
+def create_api() -> MockPrApi:
+    return MockPrApi()
+
+
+def create_config() -> V1:
+    cfg = V1(version=1)
+    cfg.merge.automerge_label = "automerge"
+    cfg.merge.blacklist_labels = []
+    cfg.merge.method = MergeMethod.squash
+    return cfg
+
+
+class MergeableType(Protocol):
+    """
+    A type we define so our create_mergeable() can be typed.
+    """
+
+    async def __call__(
```

Comment:
I learned that this is the way to define the function type. `Callable` doesn't have enough detail.
https://mypy-lang.blogspot.com/2018/09/mypy-0630-released.html